client/src/page/Template/core/algorithms/FieldBindingManager.ts

```
@@ -0,0 +1,366 @@
+
+    // Type validation
+    if (value !== null && value !== undefined && value !== '') {
+      switch (definition.type) {
+        case 'number':
+          if (isNaN(Number(value))) {
+            errors.push('Giá trị phải là số');
+          }
+          break;
+        case 'email':
+          const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+          if (!emailRegex.test(String(value))) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)